test/integration/targets/any_errors_fatal/on_includes.yml

```diff
@@ -4,4 +4,4 @@
   hosts: testhost,testhost2
   any_errors_fatal: True
   tasks:
-     - include: test_fatal.yml
+     - include_tasks: test_fatal.yml
```

Comment:
We should not blanket change them all to `include_tasks` as it is not the same thing.

 See my PR, i changed them to the 'underlying action include took', in this case,`import_tasks`.

Reply:
Amusingly enough, this was directly copied from your change, but I have since changed it to use `import_tasks` as you suggested.  :)